test/formulas/index.js

```diff
@@ -16,40 +17,77 @@ const convertJSONArray = (array) => {
     }
 }
 const test = function(testName){
-    const date = new Date(2000, 1, 1)
-    let data = json[testName]["sheet"],
-        formula = json[testName]["formula"],
-        expected = json[testName]["expectedValue"]
-        clock =  sinon.useFakeTimers(date.getTime());
-    
-    if(data)
-        convertJSONArray(data)
-    convertJSONArray(expected)
-    
-    const parser = new FormulaParser({
-        onCell: ref => {
-            return data[ref.row - 1][ref.col - 1]
-        },
-        onRange: ref => {
-            const arr = [];
-            for (let row = ref.from.row - 1; row < ref.to.row; row++) {
-                const innerArr = [];
-                for (let col = ref.from.col - 1; col < ref.to.col; col++) {
-                    innerArr.push(data[row][col])
-                }
-                arr.push(innerArr);
-            }
-            return arr;
-        },
-        onVariable: name => {
-            return json[testName]["variables"][name]
-        },
-    });
+    let expected = json[testName]["expectedValue"]
+    if(json[testName]["convertExpected"] === true)
```

Comment:
Why not just check truthy?

Reply:
good point, I figured people would just use true

Reply:
I will change it to if(json[testName]["convertExpected"] == true)